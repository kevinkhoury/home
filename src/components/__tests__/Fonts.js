import React from 'react'
import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'

import Fonts from '../Fonts'

const fontList = ['muli', 'lora', 'proza', 'rubik', 'popp']

describe('Fonts', () => {
  fontList.forEach((font) => {
    it(`renders ${font} correctly`, () => {
      const tree = renderer.create(<Fonts fontScheme={font} />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })

  it('fails on invalid input', () => {
    expect(() => {
      render(<Font />)
    }).toThrow()
    expect(() => {
      render(<Font fontScheme="foobar" />)
    }).toThrow()
  })
})
