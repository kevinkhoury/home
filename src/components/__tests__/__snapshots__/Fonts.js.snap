// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fonts renders lora correctly 1`] = `null`;

exports[`Fonts renders muli correctly 1`] = `null`;

exports[`Fonts renders popp correctly 1`] = `null`;

exports[`Fonts renders proza correctly 1`] = `null`;

exports[`Fonts renders rubik correctly 1`] = `null`;
